email/reporter_test.go

```diff
@@ -247,6 +247,9 @@ func testReporter(t *testing.T) {
 	if reported.ReportedAt == (time.Time{}) {
 		t.Fatal("unexpected reported at", reported.ReportedAt)
 	}
+	if reported.ReportedBy != "eu-pol-2.siasky.net" {
```

Comment:
can you also check that the other ones are set?

Reply:
done